server/utilities/audio/koala-filter.mdx

```diff
@@ -0,0 +1,93 @@
+---
+title: "KoalaFilter"
+description: "Audio noise reduction filter using Koala AI technology from Picovoice"
+---
+
+## Overview
+
+`KoalaFilter` is an audio processor that reduces background noise in real-time audio streams using Koala Noise Suppression technology from Picovoice. It inherits from `BaseAudioFilter` and processes audio frames to improve audio quality by removing unwanted noise.
+
+To use Koala, you need a Picovoice access key. Get started at [Picovoice Console](https://console.picovoice.ai/signup).
+
+## Installation
+
+The Koala filter requires additional dependencies:
+
+```bash
+pip install pipecat-ai[koala]
+```
+
+You'll also need to set up your OpenAI API key as an environment variable: `KOALA_ACCESS_KEY`
+
+## Constructor Parameters
+
+<ParamField path="access_key" type="str" required>
+  Picovoice access key for using the Koala noise suppression service
+</ParamField>
+
+## Input Frames
+
+<ParamField path="AudioRawFrame" type="Frame" required>
+  Raw audio data for noise reduction processing
+</ParamField>
```

Comment:
`AudioRawFrame` are not input frames actually. The filter is called directly from the transport. It's actually not a processor.